Makefile

```diff
@@ -236,12 +236,14 @@ endif
 
 lint:
 ifneq ($(SKIP_TESTS), 1)
+	@$(MAKE) shell CMD="-c 'hack/verify-lint-config.sh'"
 	@$(MAKE) shell CMD="-c 'hack/lint.sh'"
 endif
 
-local-lint:
+lint-fix:
```

Comment:
Since its inside an `if`, I think we will write less code by being explicit here instead of writing a new command `verify-config` and adding it to the 2 lint commands

From what I am familiar (example https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4/Makefile#L55) `fmt` command is commonly used to call `go fmt`. Since `golangci-lint` does more than that, and the non fix command already contains `lint` in its name, I think `lint-fix` would make more sense here